apps/web/playwright/booking-pages.e2e.ts

```diff
@@ -153,6 +153,13 @@ testBothFutureAndLegacyRoutes.describe("pro user", () => {
     await expect(page).toHaveURL(new RegExp(`${pro.username}/${eventType.slug}`));
   });
 
+  test("it returns a 404 when a requested event type does not exist", async ({ page, users }) => {
+    const [pro] = users.get();
+    const unexistingPageUrl = new URL(`${pro.username}/invalid-event-type`, WEBAPP_URL);
+    const response = await page.goto(unexistingPageUrl.href);
```

Comment:
@keithwillcode you had `waitForResponse` here instead of a `goto` the first one doesn't actually navigates to the desired page. Hence the timeout in the tests.

Reply:
@zomars thanks for fixing. I had noticed the same. Just didn’t have time to come back around to it to fix. 🙏🏼